Changed game logic.

AmandaCirverius opened the pull request:
The previous game logic did not work fully.
The board class logic has now been much simplified and
works like a charm!
ControlCenter now renamed to more suitable Game.

Also added JavaDocs.

AmandaCirverius committed:
Changed game logic.

The previous game logic did not work fully.
The board class logic has now been much simplified and
works like a charm!
ControlCenter now renamed to more suitable Game.

Also added JavaDocs.

froderik commented on src/main/java/Board.java:
```diff
@@ -55,187 +66,99 @@ public int getBottomRow()
+     * @param move, the current move from the player.
+     * @return true if the move connected four in a row.
+     */
+    public boolean checkIfMoveWon(final Move move)
```
Det här ser mycket bättre ut nu! Men det är fortfarande en för lång metod med en held del kod som ser ut att gå att återanvända. Prova att bryta ut delar till egna metoder!

froderik commented on src/main/java/Game.java:
```diff
@@ -1,29 +1,62 @@
+
+        System.out.println("Choose column 0-5 to place pebble");
+
+        // Game loop
```
Snyggt! Kanske det kan vara en metod som heter `gameLoop` i stället för kommentaren?

froderik reviewed (approved):
Bra steg framåt! Jag väljer nu `Approve` så kan du själv välja att merga in grenen till `master` direkt eller göra föreslagna ändringar på den här grenen.

AmandaCirverius committed:
Minor refactoring

The game loop is now in a method in Game class.
Game logic in Board class now split to separate methods.
